compiler/plugins/target/ROCM/ROCMTargetUtils.cpp

```diff
@@ -6,6 +6,10 @@
 
 #include "compiler/plugins/target/ROCM/ROCMTargetUtils.h"
 
+#include "compiler/plugins/target/ROCM/builtins/ukernel/iree_uk_amdgpu_gfx1030.h"
+#include "compiler/plugins/target/ROCM/builtins/ukernel/iree_uk_amdgpu_gfx1100.h"
+#include "compiler/plugins/target/ROCM/builtins/ukernel/iree_uk_amdgpu_gfx90a.h"
+#include "compiler/plugins/target/ROCM/builtins/ukernel/iree_uk_amdgpu_gfx942.h"
```

Comment:
Could group these into a single header file, perhaps with the contents of that header file generated by the build system.

Reply:
There is by construction one header file per `c_embed_data` library. So the question becomes whether to group these 4 `c_embed_data` libraries into one. I have a specific reason not to, which will become apparent in the next PR:  I want the `c_embed_data` library for a given target architecture to contain exactly the ukernels that are supported on that architecture, so that just by looking up the table of contents, LowerToUkernels will be able to tell whether a ukernel lowering should succeed or fail.